joblib/test/test_numpy_pickle.py

```diff
@@ -547,10 +550,6 @@ def test_joblib_compression_formats(tmpdir, compress, cmethod):
                 numpy_pickle.dump(obj, dump_filename,
                                   compress=(cmethod, compress))
             excinfo.match(msg)
-        elif cmethod == 'lz4' and with_lz4.args[0]:
-            # Skip the test if lz4 is not installed. We here use the with_lz4
-            # skipif fixture whose argument is True when lz4 is not installed
-            raise SkipTest("lz4 is not installed.")
```

Comment:
I am not sure this is right. With `with_lz4`, you will skip all compressors in `_COMPRESSORS` whereas we just want to skip `cmethod=='lz4'`.
Why do you need to change this?

Reply:
I was testing different things. But you are right and I reverted this change.